tools/SourceKit/tools/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ include_directories(
 
 add_swift_lib_subdirectory(sourcekitd)
 add_swift_tool_subdirectory(sourcekitd-test)
-if(LibEdit_FOUND)
+if(LibEdit_HAS_UNICODE)
```

Comment:
Similar, could you use `LibEdit_FOUND AND LibEdit_HAS_UNICODE`?

Reply:
Sure.